core/src/main/java/jenkins/model/GlobalUIConfiguration.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * The MIT License
+ * 
+ * Copyright (c) 2012, Dominik Bartholdi
```

Comment:
How bad is to copy and paste :man_facepalming: 
```suggestion
 * Copyright (c) 2020
```